boards/st/nucleo_h745zi_q/doc/index.rst

```diff
@@ -112,6 +112,8 @@ features:
 +-------------+------------+-------------------------------------+
 | SPI         | on-chip    | spi                                 |
 +-------------+------------+-------------------------------------+
+| FDCAN       | on-chip    | CAN-FD Control Area Network         |
```

Comment:
A note in the doc explaining the conflict and disabled state by default would be nice (as some people don't check dts files)

Reply:
I will change to pins set mentioned by @FRASTM and enable it then.